plugins/org.jkiss.dbeaver.model/plugin.xml

```diff
@@ -181,7 +181,7 @@
                 custom="true">
             <type kind="NUMERIC"/>
             <propertyGroup label="Properties">
-                <property id="unit" label="Unit" type="string" description="%EpochTimeTransformer.property.unit.description" defaultValue="milliseconds" required="true" validValues="seconds,milliseconds,nanoseconds,dotnet,w32filetime,oadate,sqliteJulian"/>
+                <property id="unit" label="Unit" type="string" description="%EpochTimeTransformer.property.unit.description" defaultValue="MILLISECONDS" required="true" validValues="SECONDS,MILLISECONDS,MICROSECONDS,NANOSECONDS,DOTNET,W32FILETIME,OADATE,SQLITEJULIAN"/>
```

Comment:
We cannot rename constants because they are saved in preferences. All existing configurations will be broken.

Reply:
![image](https://user-images.githubusercontent.com/22797528/229800668-e2aa1bc2-55a0-4bf7-ac76-e1dbfb071dbf.png)

Reply:
Please ignore it here. This code were created before any checks appear in the project, but modifying it now will broke backward compatibility.